package.json

```diff
@@ -32,8 +35,11 @@
     "start:affinidi-django-authlib": "python3 samples/affinidi-django-authlib/manage.py migrate && python3 samples/affinidi-django-authlib/manage.py runserver 8000",
     "start:auth0-django-authlib": "python3 samples/auth0-django-authlib/manage.py migrate && python3 samples/auth0-django-authlib/manage.py runserver 8001",
     "start:affinidi-flask-authlib": "cd samples/affinidi-flask-authlib && python3 -m flask run -p 8100",
+    "start:auth0-flask-authlib": "cd samples/auth0-flask-authlib && python3 -m flask run -p 8101",
     "start:affinidi-fastapi-authlib": "cd samples/affinidi-fastapi-authlib && python3 app.py",
```

Comment:
what are port numbers for fastapi and startlette? Are they fixed?

Reply:
yes, they are fixed with 8201 and 8301 respectively.